Newapi

yuxingfengs opened the pull request:
add function in vue and add illustration in read me

yuxingfengs committed:
add function

yuxingfengs committed:
function

yuxingfengs committed:
Merge branch 'newapi' of https://github.com/Atraxus/etherpledge into newapi

yuxingfengs committed:
Update README.md

Atraxus commented on README.md:
```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
+   - Network Name: ETH
+   - New RPC URL: http://127.0.0.1:8545
+   - Chain ID: 1337
+3. Import an account from Ganache into MetaMask.
```
Points one through three are a little unspecific. I'm not sure what to do here. We can discuss this further in the meeting

Atraxus commented on README.md:
```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
+In the terminal of the Visual Studio Code console, follow these steps:
+
+1. Install Vue CLI globally: `npm install -g @vue/cli`
+2. Install web3: `npm install web3`
```
Packages that are not installed globally should be installed through `npm install` via the `packages.json` file

Edit: web3 and big-integer are already covered in the `packages.json`, so instructions 2. and 3. are unnecessary, and instruction 1. is already in the **Prerequisites** section.

Atraxus commented on README.md:
```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
 - Run ``npm run serve`` and open the displayed URL in your browser
 
 
+## Project Setup
```
This section's instructions are already covered in the **Backend** section

Atraxus commented on README.md:
```diff
@@ -22,3 +22,34 @@ Run the following commands in the project directory:
+   - Chain ID: 1337
+3. Import an account from Ganache into MetaMask.
+
+## Start Vue
```
This section is also already covered. We could add the `npm run build` command to the front end, though.

Atraxus commented on src/components/FirstVue.vue:
```diff
@@ -51,55 +49,58 @@ export default {
+        if (typeof window.ethereum !== 'undefined') {
           this.web3 = new Web3(window.ethereum);
+
+          // 等待用户登录和选择账户
```
Please keep comments in english

yuxingfengs committed:
add router login

yuxingfengs committed:
Update README.md

yuxingfengs committed:
add direct to Creat Projetc

nwuzb committed:
Update README.md

add the way to install Vant UI ,inorder to load and use the Button of the Usercenter Informations Page.

Atraxus committed:
Fixing the pledge function

Atraxus committed:
Merge remote-tracking branch 'refs/remotes/origin/newapi2'

Conflicts:
	package-lock.json
	package.json
	src/components/FirstVue.vue
	src/components/UserCenter.vue

Atraxus committed:
Scanning past events and event subscriptions

yuxingfengs committed:
change create crowdfunding

Atraxus committed:
Main campaign is now Etherpledge

nwuzb committed:
Change the Chinese comment of ‘UserCenter.vue' to English comment

nwuzb committed:
Supplement and modify comment language to English

nwuzb committed:
Change the style of Usercenter to the display style of the computer browser (previously adapted to the mobile phone inphone XR)

Atraxus committed:
Experiment with splitting front and backend

Atraxus committed:
Getting everything running again

Atraxus committed:
major rework WIP

Atraxus committed:
Contract deployment and db

Atraxus committed:
Loading main campaign from db

Atraxus committed:
More info from the db

Atraxus committed:
Creating campaign works

Atraxus committed:
List of campaigns

Atraxus committed:
Loading raised funds from events. Reload needed

johannes-ottliczky committed:
Merge branch 'split-front-back' into newapi
